app/policies/submission_policy.rb

```diff
@@ -19,7 +19,7 @@ def manage_files?
   end
 
   def manage_subdirectories?
-    role.instructor? || role.ta?
+    user.end_user?
```

Comment:
Eh, it seems like this policy is no longer necessary. You can clean up the code a bit by removing its uses, and the corresponding `enableSubdirs` prop from the file browser React component(s).

Reply:
We talked about this in the developer meeting last Wednesday... we would like to keep this in place because I would like to make this an assignment option in the future (allow subdirectories or no). 

@pretendWhale This does remind me though that we should be checking this policy in the `update_files` method to ensure that the current user is allowed to create directories

Reply:
Good point, sounds good to me @mishaschwartz!

@pretendWhale could you please also make a new issue for adding an assignment option to allow subdirectory creation/deletion, and link it to this PR?